docker-entrypoint.sh

```diff
@@ -59,7 +34,7 @@ ROOTFS="/usr/code/rootfs/rootfs.img"
 DOCKERFS="/usr/code/rootfs/dockerfs.img"
 KUBELETFS="/usr/code/rootfs/kubeletfs.img"
 MAC_ADDRESS=$(printf 'DE:AD:BE:%02X:%02X:%02X\n' $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)))
-
+IP_ADDRESS=$(ip addr show dev eth0 | grep inet | awk '{print $2}' | cut -d \/ -f1)
```

Comment:
It could be that this expression yields multiple lines of output which might cause unexpected linefeed to be injected on `qemu` execution below. We were experience problem on `geckon` with this when the error message was that no DNS servers was provided, even though they were and everything was good.

Reply:
sure, this is mostly my trying stuff, when i have  working solution we might improve this line

any better ideas how to fetch the ip from the interface more reliable?

Reply:
> any better ideas how to fetch the IP from the interface more reliable?

`ip -j addr show | jq '<put magical json path that extracts correct ip of eth0>'`